### ISSUE TITLE
Creative software not working.

### ISSUE THREAD

**Issue by osctorand** [AlanFinotty1995/AAF_DCH_RealtekModded#7]
The creative programs does not work with this driver. The program simply says "Your audio device cannot be detected."

![error](https://user-images.githubusercontent.com/55161298/88469136-2c5c3480-ceee-11ea-83aa-3e891634b413.PNG)

**Comment by RCCore** [comment]
I have a question do you install Sound Blaster Cinema 5 or 720?

**Comment by RCCore** [comment]
What motherboard are you using and what codec do you have?

**Comment by osctorand** [comment]
I am using an OEM motherboard from an acer prebuilt, The model number is DG.E0MEQ.10S. I am also using default windows codecs. I am also installing Sound blaster 720.

**Comment by AlanFinotty1995** [comment]
Check the mandatory spoiler on the TechPowerUp thread IN [**THIS LINK**](https://www.techpowerup.com/forums/threads/realtek-modded-audio-driver-for-windows-7-8-1-10-dch-uad-version.250915/).

**Comment by osctorand** [comment]
I will see if this works. If not, I will get back to you. Thank you so far!

Den ons 5 aug. 2020 00:12Alan Finotty <notifications@github.com> skrev:

**Comment by osctorand** [comment]
Hello again. I tried the solutions, and even uninstalled the driver and
reinstalled the newest version, but it is still not working.

**Comment by osctorand** [comment]
I am also wondering how to revert / uninstall the AAF optimus bus driver,
since I think it might be causing trouble with my audio. Is there a way to
do that?

**Comment by RCCore** [comment]
@osctorand yes it is a way to uninstall it just download driver store explorer and search for **AAF Bus Driver** and select force delete.

**Comment by epman100** [comment]
get the latest version of DriverStore Explorer from [here](https://github.com/lostindark/DriverStoreExplorer/releases) and follow what RCCore said
